src/env.h

```diff
@@ -980,6 +980,8 @@ class Environment : public MemoryRetainer {
 
   void PrintSyncTrace() const;
   inline void set_trace_sync_io(bool value);
+  inline bool get_still_synchronous();
```

Comment:
Fwiw, getters in the C++ code base (both Node and V8 parts) aren’t usually prefixed with `get_`